app/services/activation/activation_service.rb

```diff
@@ -63,33 +63,39 @@ def prospective?
         ProspectiveEmployee.not_archived.exists?(id_number: id_number)
     end
 
+    def send_verification_code?
+      Twilio::Verify.send_phone_verification_code(mobile_phone) == 'ok'
+    end
+
     def active
       return unless valid?
```

Comment:
Bu `valid?` metodu nerede? Ve neden `StandardError` donme ihtimali var?

Reply:
[valid? metodu](https://guides.rubyonrails.org/active_record_validations.html#valid-questionmark-and-invalid-questionmark). Gerçekleme yaparken bazen `RuntimeError` ile karşılaştığım olmuştu. Başka hangi hata ile karşılaştığımı şu an hatırlamıyorum. Session mekanizması ile düzenleme yaparken burayı tekrar gözden geçiririm.